commune_app/migrations/0002_add_user_test_data.py

```diff
@@ -0,0 +1,27 @@
+from django.db import migrations, transaction
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('commune_app', '0001_initial'),
+    ]
+
+    def generate_user_test_data(apps, schema_editor):
+        from ..all_models.users import User
+
+        users_test_data = [
+            ('testUser123', 'testUser1@mta.ac.il', 'PasswordU$er123', '123'),
```

Comment:
Please change it like this: 
```suggestion
            ('testUser123', 'testUser1@telhai.ac.il', 'PasswordU$er123', '123'),
```